portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,46 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Ibiyemi's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <header>
+    <h1>
+      Ibiyemi Abiodun
+    </h1>
+  </header>
+  <section id="content">
+    <h2>
+      Hi, I'm Ibiyemi Abiodun!
+    </h2>
+    <p>
+      I'm a rising sophomore at Cornell, double majoring in electrical &
+      computer engineering, and computer science. I love cooking & baking,
+      karate, VFX, cinematography, and of course, coding.
+    </p>
+    <p>
+      I was born in
+      Nigeria, and I've lived all over the U.S., although when people ask I tell them I'm from Seattle because I lived in Bellevue & Kirkland for about 9 years of my 19-year life.
+    </p>
+    <h2>Career</h2>
+    <p>
+      You want my resume? <a href="/dl/resume.pdf">Here you go.</a>
+    </p>
+    <h2>Cooking</h2>
+    <p>
+      <!-- TODO -->
+    </p>
+  </section>
+  <footer>
+    <p>&copy; 2020 Google, Inc.</p>
+    <p>Because this stuff isn't owned by <em>me</em>...</p>
+  </footer>
+
+</body>
+
+</html>
```

Comment:
It looks like there needs to be a newline at the end of this file.

Reply:
Sure, but why?

Reply:
It's a pretty common best practice to require a newline to be the last character in a file. This helps keep the `git blame` cleaner so that devs only modify lines they actually changed. So they wouldn't be changing line 46 to add content after it. It's the same reason folks will require commonly modified arrays to have each element on its own line with a trailing comma (even the last one):
```
const vals = [
  1,
  2,
  3,
];
```
```

Reply:
Got it, thanks! I fixed this.